scripts/create_burndown.py

```diff
@@ -26,12 +26,16 @@ def parseCommandLine():
   parser = argparse.ArgumentParser(epilog=epilog, description=description)
   parser.add_argument('sprint', metavar='NUM', help='Sprint Number')
   parser.add_argument('--output', help='Location of data to process')
+  parser.add_argument('--location', help='Image file location')
   parser.add_argument('--no-tasks', action='store_true', help='Disable Tasks line in the chart', default=False)
   parser.add_argument('--with-fast-lane', action='store_true', help='Draw line for Fast Lane cards', default=False)
   parser.add_argument('--verbose', action='store_true', help='Verbose Output', default=False)
   parser.add_argument('--no-head', action='store_true', help='Run in headless mode', default=False)
   args = parser.parse_args()
 
+  if args.location:
+    args.location = os.path.join(os.getcwd(),args.location)
```

Comment:
what about doing this inside `saveImage`? It is actually not needed at this point. 😉

Reply:
Actually, when we pass --output=' ' for indicating the data file we switch to that directory at the very next line of code and saveImage is called in the last so when saveImage is called the current directory would be 'output' directory and not the directory the user is in now so I updated this here. :smile: